MediaBrowser.Providers/Lyric/LrcLyricParser.cs

```diff
@@ -68,8 +68,11 @@ public LrcLyricParser()
 
         for (int i = 0; i < sortedLyricData.Count; i++)
         {
+            List<TimeTag> timeTags = sortedLyricData[i].TimeTags
+                .Select(kvp => new TimeTag(position: kvp.Key.Index + (kvp.Key.Index == -1 ? 1 : 0), timestamp: kvp.Value * 10000)).ToList();
```

Comment:
I find this hard to parse, maybe change it to be 

```c#
position: Math.Max(kvp.Key.Index, 0)
```

Would also change this to only create the list of there are any TimeTags on the parsed data

```c#
List<TimeTag>? timeTags = null;
if(sortedLyricData[i].TimeTags.Count != 0)
{
	// TODO build list here
}
```

Reply:
I've changed it to use that instead